jdaviz/configs/default/plugins/tools.py

```diff
@@ -16,11 +16,8 @@ def __init__(self, *args, **kwargs):
         self._data = None
 
     def _reset_profile_viewer_bounds(self):
-        pv_state = self._profile_viewer.state
-        pv_state.x_min = self._previous_bounds[0]
```

Comment:
Not sure why originally @rosteen decided to also reset X axis in https://github.com/spacetelescope/jdaviz/pull/2647 . Maybe he can clarify in case I overlooked something.